scripts/pkglib.sh

```diff
@@ -0,0 +1,314 @@
+#!/bin/bash
+
+# This is a helper library for the setup scripts of each package
+# in this repository.
+
+source ../versions.txt
+PACKAGING_DIR=/var/packaging
+LOG_DIR=${PACKAGING_DIR}/build_logs
+
+# BUILD OPTIONS
+BUILD_DISTROS=${BUILD_DISTROS:-Fedora_27 xUbuntu_16.04 CentOS_7)
+BUILD_ARCH=${BUILD_ARCH:-}x86_64
+
+function display_help()
+{
+	cat <<-EOL 
+	$SCRIPT_NAME
+
+	This script is intended to create Kata Containers 3.X packages for the OBS 
+	(Open Build Service) platform.
+
+    Usage:
+        $SCRIPT_NAME [options]
+
+	Options:
+
+    -l         --local-build     Build the runtime locally
+    -w	       --workdir         Repository directory
+    -c         --commit-id       Build with a given commit ID
```

Comment:
this is the commit id to the for the project to be packaged?  If I change this will move to that commit but keep the version we define that should be the expected version of pkg generated?